extensions/quick-web/src/edit_arg_value.tsx

```diff
@@ -0,0 +1,47 @@
+import { Action, ActionPanel, Form, popToRoot } from "@raycast/api";
+import { argValueExists, updateArgValue } from "./arg_value_repository";
+import { useState } from "react";
+
+export default function EditArgValueCommand({ argName, oldArgValue }: { argName: string; oldArgValue: string }) {
+  const [argValue, setArgValue] = useState<string>(oldArgValue);
+  const [argValueErr, setArgValueErr] = useState<string | undefined>("");
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm
+            onSubmit={() => {
+              console.log("edit arg val: ", argValue);
+              updateArgValue(argName, oldArgValue, argValue).finally();
+              popToRoot().finally();
+            }}
+          />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField
+        id="newArgValue"
+        title={"Arg Value"}
+        value={argValue}
+        error={argValueErr}
+        onChange={async (newValue) => {
+          setArgValue(newValue);
+
+          if (newValue.length == 0) {
+            setArgValueErr("The field shouldn't be empty!");
+            return;
+          } else {
+            setArgValueErr(undefined);
+          }
+
+          if (await argValueExists(argName, newValue)) {
```

Comment:
```suggestion
          if (argValue !== newValue && (await argValueExists(argName, newValue))) {
```
Could maybe be like this?

Reply:
If i didn't get it wrong, do you mean I handle it in the code after submission? If it is true, it is not the block way I need. I need to block the submission

Reply:
What about if it's the same name just add a if sentence that just don't run the code if it's the same input?

Reply:
> What about if it's the same name just add a if sentence that just don't run the code if it's the same input?

I feel it is consufing for the users. It is like a data manage system, I think the users should always have a way to explicitly know how the data is like in the system. Otherwise it is like a black box to the users.

Reply:
Is the modification mandotory? Crying @pernielsentikaer

Reply:
No, I don't think, but it would be good to handle in a later update 🙂